vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -17,70 +17,44 @@ namespace winrt {
   using namespace Windows::UI::Xaml::Interop;
 }
 
+static const std::map<std::string, winrt::FlyoutPlacementMode> placementModeMinVersion = {
```

Comment:
may want to use unordered_map as slightly cheaper.  don't think either can be constexpr unfortunately